src/errors.ts

```diff
@@ -63,6 +63,16 @@ export const SubmissionFailErrorMessage: Record<
   ONBOARDING_TAG_LIMIT_REACHED: 'Tag limit reached',
 };
 
+export enum SourceRequestErrorKeys {
+  AccessDenied = 'ACCESS_DENIED',
+}
+
+export const SourceRequestErrorMessage: Record<SourceRequestErrorKeys, string> =
+  {
+    ACCESS_DENIED:
```

Comment:
```suggestion
    [SourceRequestErrorKeys.AccessDenied]:
```

Reply:
Added in bf84cac2012c4203be2757fe38f619fe9f9931ad